src/components/dataTile/DataTile.scss

```diff
@@ -0,0 +1,63 @@
+@use '../../styles/colors' as colors;
+@use '../../styles/fonts' as fonts;
+
+.tile {
+  flex: 1;
+  background-color: #fff;
+  border: 1px solid #e3e6f0;
+  border-left: none;
+  border-radius: 0.35rem;
+  padding: 1.25rem;
+  box-shadow: 0 .15rem 1.75rem 0 rgba(58,59,69,.15);
+  position: relative;
+  min-width: 12em;
+  p{
+    color: colors.$dark;
+    font-weight: 500;
+    margin: 0;
+  }
+  p:first-child{
+    text-transform: uppercase;
+    font-size: 0.5rem;
+  }
+  &-green {
+    p:first-child{
+      color: colors.$green;
+    }
+  }
+  &-teal {
+    p:first-child{
+      color: colors.$teal;
+    }
+  }
+  &-blue {
+    p:first-child{
+      color: colors.$primary;
+    }
+  }
+  &-yellow {
+    p:first-child{
+      color: colors.$yellow;
+    }
+  }
+}
+
+.tile::before{
```

Comment:
to możesz wrzucić wyżej do klasy .tile {
&::before
}
z resztą pseudo elementów tez tak zrob

Reply:
Z początku tak chciałem ale nie działało i zrobiłem jeszcze raz i nagle działa
więc jest ok